src/components/Details.js

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import { v4 as uuidv4 } from 'uuid';
+import { showDetails } from '../redux/statisticsSlice';
+import '../App.css';
+
+const Details = () => {
+  const dispatch = useDispatch();
+
+  const { id } = useParams();
+
+  useEffect(() => {
+    dispatch(showDetails(id));
+  });
+
+  const { dataArray } = useSelector((state) => state.Covid);
+
+  const filteredStats = dataArray.filter((data) => data.show);
+
+  return (
+    <div className="detailsContainer">
+      {
+        filteredStats.map((data) => (
+          <div key={uuidv4()} className="detailContainer">
+            <h1>
+              Country:
+              { ' ' }
+              { data.country }
+              ,
+              { ' ' }
+              { data.region }
+            </h1>
+            <h2>
+              New Cases:
+              {' '}
+              { data.cases.new }
+              ,
+            </h2>
```

Comment:
- [ ] Great job working on the details page but I think it would be great if you make the details layout as the given design since it is different so please make it the same so when you are getting the information you can align it the name in the left and the number in the right take a look to the images here 🙏🏻🔥🤩

- [ ] Please add the section in the middle as well with an image and the name of the country in the side so that will improve from the user experience and to match the given design 🤩🔥💯

- Your
![image](https://github.com/dazehinn/world-covid-19-data/assets/58553711/ac90a4d5-6d6b-4e02-8140-4204b3654ccb)



- Given
![image](https://github.com/dazehinn/world-covid-19-data/assets/58553711/f0d14104-f1ea-47d2-befa-b9619baf3d57)